### PR TITLE
Javascript/javascript1/week2

### PR DESCRIPTION
**Pull request by Saidemm**

### CODE REVIEW

File: javascript/javascript1/week2/homework/event-application.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+function getEventWeekday(dayFromToday) {
```
Comment (flom84) [inline javascript/javascript1/week2/homework/event-application.js:2]: if dayFromToday is 1 then this function doesn't return the correct day. can you spot the issue?

File: javascript/javascript1/week2/homework/student-manager.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName == null || studentName == "") {
+        console.log("Please fill the name correctly");
+    }
+    else if (getNumberOfStudents() >= 6 && studentName != "Queen") {
```
Comment (flom84) [inline javascript/javascript1/week2/homework/student-manager.js:6]:
```suggestion
    else if (getNumberOfStudents() > 6 && studentName != "Queen") {
```

File: javascript/javascript1/week2/homework/student-manager.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName == null || studentName == "") {
```
Comment (flom84) [inline javascript/javascript1/week2/homework/student-manager.js:3]:
```suggestion
    if (studentName === null || studentName === "") {
```

File: javascript/javascript1/week2/homework/student-manager.js (hunk 2 above)
Comment (Saidemm) [reply javascript/javascript1/week2/homework/student-manager.js:6]: Actually, I think = is necessary here, as the class should hold max 6 students (unless it's the Queen). And at this point, the new student is not yet added, so if the current size is 6, the class is full, and an error should be displayed.

File: javascript/javascript1/week2/homework/student-manager.js (hunk 3 above)
Comment (Saidemm) [reply javascript/javascript1/week2/homework/student-manager.js:3]: I'm afraid I have to disagree, as == covers more cases of possible invalid inputs e.g. studentName == null covers both undefined and null values.

File: javascript/javascript1/week2/homework/event-application.js (hunk 1 above)
Comment (Saidemm) [reply javascript/javascript1/week2/homework/event-application.js:2]: Thanks for the input. An extra % was missing :)
I will fix and push

**Review by flom84 — APPROVED** [review]

**Pull request merged**